pkg/config/plugins/runtime/k8s/config.go

```diff
@@ -97,8 +97,10 @@ type Injector struct {
 	VirtualProbesEnabled bool `yaml:"virtualProbesEnabled" envconfig:"kuma_runtime_kubernetes_virtual_probes_enabled"`
 	// VirtualProbesPort is a port for exposing virtual probes which are not secured by mTLS
 	VirtualProbesPort uint32 `yaml:"virtualProbesPort" envconfig:"kuma_runtime_kubernetes_virtual_probes_enabled"`
-	// Cnfiguration for a traffic that is intercepted by sidecar
+	// Configuration for a traffic that is intercepted by sidecar
```

Comment:
I know that you just fixed a typo, but let's make it according to the guide for comments in Go - `// SidecarTraffic is a configuration ...`